tests/node-e2e/app.js

```diff
@@ -28,13 +29,18 @@ app.put("/", (req, res) => {
         throw err;
       }
     });
+    fs.writeFile(deletePath, data.toString(), (err) => {
```

Comment:
Instead of having two separate paths, my idea was also to rather have a watch on a single file through inotify 
PUT -> CREATE 
DELETE -> REMOVE
so we won't need sleeps. but I am okay with this too, as it kinda prevents things from getting overcomplicated. 
This should close #99